module/zfs/noload.c

```diff
@@ -37,6 +38,11 @@ int nvme_algo_run(struct nvme_algo *alg, struct bio *src,
 struct nvme_algo *nvme_algo_find(const char *algo_name, const char *dev_name);
 void nvme_algo_put(struct nvme_algo *alg);
 
+struct bio_pad_data {
+	void orig, bounce;
```

Comment:
void *?... I didn't think raw void would even compile...

Reply:
I pushed a bit early, fixed this an a few others... seems void compiles to int

Reply:
Normally gcc would give you this error:

```
<source>:5:10: error: variable or field 'blah' declared void
    5 |     void blah;
      |          ^~~~
```

But ZFS must not have the compile strict enough...